includes/class-sensei-utils.php

```diff
@@ -292,12 +292,38 @@ public static function sensei_text_editor( $content = '', $editor_id = 'senseite
 			'tinymce'       => array(
 				'theme_advanced_buttons1' => $buttons,
 				'theme_advanced_buttons2' => '',
+				'setup'                   => 'function (editor) {
```

Comment:
Could we keep this JS code in a separate file and use `file_get_contents` or similar to ease future maintenance?

Reply:
I couldn't use `file_get_contents` here because we needed to import the i18n library, but I've implemented another approach to load js file inside tinymce iFrame https://github.com/Automattic/sensei/pull/7124/commits/ab737419d5603d760f69700b4b8416dfd9473f8a . Was able to move the JS there, had to have a little bit left inline to call that.